python/src/aiconfig/default_parsers/dalle.py

```diff
@@ -51,45 +51,64 @@ def refine_image_completion_params(model_settings):
 
 
 def construct_output(image_data: Image, execution_count: int) -> Output:
+    mime_type : Optional[str] = None
+    data : Optional[str] = None
+    
+    if image_data.b64_json:
+        mime_type = "text/plain"
+        data = image_data.b64_json
```

Comment:
I don't think this is correct. Even if the data is base-64 encoded, the MIME type is for what the data represents. So I believe it should always be `image/png`. cc @rholinshead as well

Reply:
Yep, that's correct. One thing we should consider is if we want to just enforce serialized data as uris instead of supporting binary data dumped into the config. The latter will be a huge pain code review and likely for performance down-the-line (e.g. saving/loading a big json blob with many images is not going to be super fast)